src/loaders/OmeZarrLoader.ts

```diff
@@ -395,7 +395,7 @@ class OMEZarrLoader extends ThreadableVolumeLoader {
       volumeSize: pxSizeLv,
       subregionSize: pxSizeLv.clone(),
       subregionOffset: new Vector3(0, 0, 0),
-      physicalPixelSize: new Vector3(scale5d[x], scale5d[y], hasZ ? scale5d[z] : Math.min(scale5d[x], scale5d[y])),
+      physicalPixelSize: new Vector3(scale5d[4], scale5d[3], hasZ ? scale5d[2] : Math.min(scale5d[4], scale5d[3])),
```

Comment:
What was the bug that this fixes?

Reply:
if the data was TZYX then the zyx indices can be wrong (as opposed to TCZYX).  The scale5d variable is always in TCZYX order, so you can use explicit indexing.  That's why the variable has "5d" in its name and also why I added a little type hint where the variable is declared above.

Reply:
Gotcha, thanks for the explanation!